checker/tests/lock/JCIPAnnotations.java

```diff
@@ -1,95 +1,264 @@
+// @skip-test
```

Comment:
We decided to keep the JCIP functionality. An issue isn't needed since we've agreed that this must work before the Lock Checker is ready to be pushed.